patches/core/inox-patchset/0001-fix-building-without-safebrowsing.patch

```diff
@@ -227,18 +225,26 @@
    } else if (path_without_query == "/clock") {
      interstitial_delegate.reset(CreateBadClockBlockingPage(web_contents));
    } else if (path_without_query == "/lookalike") {
-@@ -498,11 +385,6 @@ void InterstitialHTMLSource::StartDataRe
+@@ -502,16 +389,12 @@ void InterstitialHTMLSource::StartDataRe
+         CreateOriginPolicyInterstitialPage(web_contents));
+   }
  
-   if (path_without_query == "/supervised_user") {
-     html = GetSupervisedUserInterstitialHTML(path);
--  } else if (path_without_query == "/quietsafebrowsing") {
+-  if (path_without_query == "/quietsafebrowsing") {
 -    TestSafeBrowsingBlockingPageQuiet* blocking_page =
 -        CreateSafeBrowsingQuietBlockingPage(web_contents);
 -    interstitial_delegate.reset(blocking_page);
 -    html = blocking_page->GetHTML();
-   } else if (interstitial_delegate.get()) {
+ #if BUILDFLAG(ENABLE_SUPERVISED_USERS)
+-  } else if (path_without_query == "/supervised_user") {
++  if (path_without_query == "/supervised_user") {
+     html = GetSupervisedUserInterstitialHTML(path);
++  }
+ #endif
+-  } else if (interstitial_delegate.get()) {
++  if (interstitial_delegate.get()) {
```

Comment:
Breaking up that conditional statement can cause different behavior. Better start it with `if (false) {` instead.

Reply:
Perhaps I should not break up that conditional statement then.